tests/ops/functions/test_simplify.py

```diff
@@ -162,3 +162,24 @@ def qfunc():
         assert s_op.data == simplified_tape_op.data
         assert s_op.wires == simplified_tape_op.wires
         assert s_op.arithmetic_depth == simplified_tape_op.arithmetic_depth
+
+    @pytest.mark.jax
+    def test_jitting_simplified_qfunc(self):
+        """Test that we can jit qnodes that have a simplified quantum function."""
+
+        import jax
+
+        @jax.jit
+        @qml.qnode(qml.device("default.qubit.jax", wires=1), interface="jax")
+        @qml.simplify
+        def circuit(x):
+            qml.adjoint(qml.RX(x, wires=0))
+            qml.PauliX(0) ** 2
+            return qml.expval(qml.PauliY(0))
```

Comment:
The error appears even without jitting, right? Is it necessary to jit in this test?

Reply:
If it works for jitting, then it will work for non-jitting. And we need to test that it works with jit.